DOTNET-example/README.md

```diff
@@ -14,23 +14,11 @@ will eventually export the trace data to our UI.
 * A VMware Aria Operations for Applications account (formerly known as Tanzu Observability by Wavefront) account, which gives you access to a cluster. If you don’t have a cluster, [sign up for a free trial](https://www.vmware.com/products/aria-operations-for-applications.html).
 * Clone the [OpenTelemetry Examples](https://github.com/wavefrontHQ/opentelemetry-examples) repository.
 * Install the Docker platform. You’ll run the Wavefront proxy on Docker for this tutorial.
```

Comment:
Are we keeping the Docker requirement for all of these examples, or removing it?

Reply:
Thank you! I missed this. I will remove it from here!